src/Directory.Solution.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="build.warnings_as_errors.props" />
```

Comment:
It is maddeningly difficult to get the same behavior for building .sln as .csproj for these settings.

Reply:
indeed!